api/serializers/teacher.py

```diff
@@ -95,3 +94,25 @@ class Meta:
             "peer_support",
         )
         read_only_fields = fields
+
+
+class PublicTeacherSerializer(CommonPublicTeacherSerializer):
+    """Representation of a Teacher that is used in 'All teachers' Tooljet view."""
+
+    class Meta(CommonPublicTeacherSerializer.Meta):
+        pass
+
+
+class PublicTeacherWithPersonalInfoSerializer(CommonPublicTeacherSerializer):
+    """Representation of a Teacher that is used in 'Teacher by coordinator' Tooljet view."""
+
+    personal_info = PublicPersonalInfoSerializer()
+    date_and_time_added = serializers.DateTimeField(source="personal_info.date_and_time_added")
+
+    # TODO LogEvent ?
+
+    class Meta(CommonPublicTeacherSerializer.Meta):
+        fields = CommonPublicTeacherSerializer.Meta.fields + (
+            "personal_info",
+            "date_and_time_added",
```

Comment:
`date_and_time_added` is not that personal. It can be viewed by all coordinators (it's useful if everybody can see how long the teacher has been with SSG)

Reply:
Do you mean, this and `non_teaching_help_required` should be put to public serializer (to CommonPublic)?

Reply:
yes :) after all, you put `non_teaching_help_provided` into the public teacher serializer, right?

Reply:
Fixed